ci/pipeline.yml

```diff
@@ -343,7 +343,7 @@ jobs:
     passed:
       - sbt-buildo
       - enumero
-      # - mailo
+      - mailo
```

Comment:
👍 I think it was commented because I was blocked by something, possibly the very same fix of this PR.